lnbits/decorators.py

```diff
@@ -6,14 +6,37 @@
 from fastapi.openapi.models import APIKey, APIKeyIn
 from fastapi.security import APIKeyHeader, APIKeyQuery
 from fastapi.security.base import SecurityBase
-from pydantic.types import UUID4
 
 from lnbits.core.crud import get_user, get_wallet_for_key
 from lnbits.core.models import User, WalletType, WalletTypeInfo
 from lnbits.db import Filter, Filters, TFilterModel
 from lnbits.requestvars import g
 from lnbits.settings import settings
 
+# from pydantic.types import UUID4
+
+
+def is_user(request: Request):
```

Comment:
- from the name of the function I expect for the return type to be a `Bool`
- this function is in fact a getter

Reply:
it is a decorator, maybe calling them require_user and so on is better?